generator/Makefile

```diff
@@ -368,3 +370,12 @@ $(MIBDIR)/readynas:
 $(MIBDIR)/readydataos:
 	@echo ">> Downloading readydataos"
 	@curl $(CURL_OPTS) -o $(MIBDIR)/readydataos "$(READYDATAOS_URL)"
+
+$(MIBDIR)/SchneiderPM55xx_V01_13.mib:
+	$(eval TMP := $(shell mktemp))
+	@echo ">> Downloading SchneiderPM55xx_V01_13.mib to $(TMP)"
+	@curl $(CURL_OPTS) -o $(TMP) $(SCHNEIDER_PM55XX_URL)
+	@unzip -j -d $(MIBDIR) $(TMP) PM5560_PM5563_v2.1.0/SchneiderPM55xx_V01_13.mib
+	# Workaround invalid character in Schneider MIB
+	@sed -i 's/_//g' $(MIBDIR)/SchneiderPM55xx_V01_13.mib
```

Comment:
Is this still necessary, given that we support mibopts (https://github.com/prometheus/snmp_exporter/pull/873) and you can simply allow those characters by adding mibopt "u" ?

Reply:
Ah, totally slipped my mind that is an option. Switched it out in favor of your idea. The `--snmp.mibopts u` works perfectly.

Reply:
Hmm, I'm not sure I'm happy with adding a MIB that requires custom mibopts. Perhaps we should adjust the generator default mibopts?

Or maybe get Schneider to fix their MIB. Cisco had some similar issues in the AIRESPACE-WIRELESS-MIB that I got them to fix. Teach them about `smilint` :grin: